sdk/src/manifest_store.rs

```diff
@@ -297,8 +297,8 @@ mod tests {
         assert!(manifest.time().is_some());
     }
 
+    #[cfg_attr(not(target_arch = "wasm32"), actix::test)]
     #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
-    #[ignore]
     #[allow(dead_code)]
     async fn manifest_report_image_wasm() {
```

Comment:
If we are going to support running this test outside of wasm, we should change the name